sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -79,7 +79,7 @@ export default {
     const PROGRAM_NOT_LISTED = "myProgramNotListed";
     const OFFERING_NOT_LISTED = "myStudyPeriodIsntListed";
 
-    let formInstance: any;
+    const formInstance = ref();
```

Comment:
Please consider keeping the _formInstance_ as a regular property and creating a computed property to deal with the visibility of the navigation buttons.
The _formInstance_ is not actually being used in the template and it is usually not needed.